PSScripts/Set-CorsOnStorageAccount.ps1

```diff
@@ -0,0 +1,69 @@
+<#
+
+.SYNOPSIS
+Adds the CORS header to storage account responses

```

Comment:
Should read "Replaces the existing CORS rules if the rule that is being set doesn't exist"